substra/sdk/client.py

```diff
@@ -520,6 +611,17 @@ def download_algo(self, asset_key, destination_folder):
         url = data['content']['storageAddress']
         self._download(url, destination_folder, default_filename)
 
+    def download_composite_algo(self, asset_key, destination_folder):
+        """Download composite algo resource.
+
+        Download composite algo package in destination folder.
+        """
+        data = self.get_composite_algo(asset_key)
+        # download composite algo package
+        default_filename = 'algo.tar.gz'
```

Comment:
Could the default name be `composite_algo.tar.gz`? It's not much of a difference but it's explicit.